src/analytics/ValoraAnalytics.ts

```diff
@@ -107,8 +108,10 @@ class ValoraAnalytics {
           }
         : null
 
+      const overrideStableID = isE2EEnv ? DEFAULT_STATSIG_ID : await Analytics.getAnonymousId()
```

Comment:
why do we need this? can you add an explanation in a code comment?

Reply:
It's unclear why, but `getAnonymousId` caused many e2e tests to fail. I tried to figure out what was going on under the hood but am still not certain. Added a brief comment